easybuild/easyconfigs/v/Vampir/Vampir-8.4.1-demo-linux-x86_64-foss-2015a.eb

```diff
@@ -0,0 +1,48 @@
+##
+# This is an easyconfig file for EasyBuild, see https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2013-2015 Juelich Supercomputing Centre, Germany
+# Authors::   Bernd Mohr <b.mohr@fz-juelich.de>
+#             Markus Geimer <m.geimer@fz-juelich.de>
+# License::   3-clause BSD
+#
+# This work is based on experiences from the UNITE project
+# http://apps.fz-juelich.de/unite/
+##
+
+easyblock='Binary'
+
+name = "Vampir"
+version = "8.4.1"
+versionsuffix = "-demo"
+variant = "linux-x86_64"
```

Comment:
You're explaining the issue yourself (sort of).

The name of the easyconfig is dictated by the combination of `name`, `version`, `toolchain` and `versionsuffix` (and `versionprefix`, which is rarely used).

So, unless you make `linux-x86_64` part of the `versionsuffix`, Jenkins will complain about the filename.

I agree this will look 'icky' in the module name; if you care enough about it, you can hide the `linux-x86_64` part via a custom module naming scheme.